cmd/cli/serve/serve.go

```diff
@@ -168,65 +164,23 @@ func serve(cmd *cobra.Command, cfg types.Bacalhau, fsRepo *repo.FsRepo) error {
 		isRequesterNode = true
 	}
 
-	networkConfig, err := getNetworkConfig(cfg)
-	if err != nil {
-		return err
-	}
-
-	computeConfig, err := GetComputeConfig(ctx, cfg, isComputeNode)
-	if err != nil {
-		return errors.Wrapf(err, "failed to configure compute node")
-	}
-
-	requesterConfig, err := GetRequesterConfig(cfg, isRequesterNode)
-	if err != nil {
-		return errors.Wrapf(err, "failed to configure requester node")
-	}
-
 	// Create node config from cmd arguments
+	// TODO: validate if this is necessary
```

Comment:
Upon inspecting the method, I don't believe this method does anything useful. Probably best to remove.